### ISSUE TITLE
datagrid rowclick / show row as link to allow right click to open context menu - open in a new tab

### ISSUE THREAD

**Issue by dkocich** [marmelab/react-admin#10139]
**Is your feature request related to a problem? Please describe.**
it would be useful to quickly open multiple items in new tabs from the LIST page

**Describe the solution you'd like**
somehow make rows appear as a classic link or allow it to use the RMB context menu - new tab; or Middle mouse button as a shortcut to open a new tab directly  

**Describe alternatives you've considered**
since the RA major update, we have used row clicks instead of "show" buttons; but recently we have been thinking to put some smaller buttons back to allow this link behaviour if that wont be possible in RA lib

**Comment by slax57** [comment]
Thank you for your suggestion.

I'm not sure this is possible. You see, contrary to `EditButton` or `ShowButton` which are simply links (we can compute) to the edit/show view, the `Datagrid` `rowClick` is a JavaScript function, as it can handle multiple cases such as `expand` or `toggleSelection`, or being passed another JS function.

**TLDR**: I don't think it is possible for technical reasons, so using `EditButton` or `ShowButton` is the way to go.
